src/fix_langevin.cpp

```diff
@@ -510,17 +643,27 @@ void FixLangevin::post_force_templated()
       if (Tp_TSTYLEATOM) tsqrt = sqrt(tforce[i]);
       if (Tp_RMASS) {
         gamma1 = -rmass[i] / t_period / ftm2v;
-        gamma2 = sqrt(rmass[i]) * sqrt(24.0*boltz/t_period/dt/mvv2e) / ftm2v;
+        if (!Tp_GJF)
```

Comment:
this would be easier to read, if the cases were swapped and the test would be `if (Tp_GJF)`.